src/conduits/java/com/enderio/conduits/common/types/ItemClientConduitData.java

```diff
@@ -16,10 +20,16 @@
 
 public class ItemClientConduitData implements IClientConduitData<ItemExtendedData> {
     @Override
-    public List<AbstractWidget> createWidgets(Screen screen, ItemExtendedData extendedConduitData, Supplier<Direction> direction, Vector2i widgetsStart) {
+    public List<AbstractWidget> createWidgets(Screen screen, Supplier<BlockPos> blockPos, Supplier<IConduitType<?>> type, ItemExtendedData extendedConduitData, Supplier<Direction> direction, Vector2i widgetsStart) {
         List<AbstractWidget> widgets = new ArrayList<>();
-        widgets.add(new CheckBox(EnderIO.loc("textures/gui/round_robin.png"), widgetsStart.add(110, 20), () -> extendedConduitData.get(direction.get()).roundRobin, bool -> extendedConduitData.compute(direction.get()).roundRobin = bool));
-        widgets.add(new CheckBox(EnderIO.loc("textures/gui/self_feed.png"), widgetsStart.add(130, 20), () -> extendedConduitData.get(direction.get()).selfFeed, bool -> extendedConduitData.compute(direction.get()).selfFeed = bool));
+        widgets.add(new CheckBox(EnderIO.loc("textures/gui/round_robin.png"), widgetsStart.add(110, 20), () -> extendedConduitData.get(direction.get()).roundRobin, bool -> {
+            extendedConduitData.compute(direction.get()).roundRobin = bool;
+            CoreNetwork.sendToServer(new C2SSetConduitExtendedData(blockPos.get(), type.get(), extendedConduitData));
```

Comment:
add todo, we can't expect addons to know about core network, nor the package

Reply:
Yeah ideally in future we pass a consumer or something that lets them sync without knowledge of core